.github/workflows/release.yml

```diff
@@ -19,12 +19,12 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version-file: 'go.mod'
-      -
-        name: Docker Login
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+#      -
```

Comment:
why is this commented but not removed?

Reply:
Left it in case we wanted to resume publishing docker builds in the future